packages/mobile/src/components/share-drawer/ShareToStorySticker.tsx

```diff
@@ -0,0 +1,111 @@
+import type { Track, User } from '@audius/common'
+import type { StyleProp, ViewStyle } from 'react-native'
+import { Image, View } from 'react-native'
+
+import AudiusLogo from 'app/assets/images/audiusLogoHorizontal.svg'
+import { Divider, Text } from 'app/components/core'
+import UserBadges from 'app/components/user-badges'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { useTrackImage } from '../image/TrackImage'
+
+const messages = {
+  by: 'by',
+  nowPlayingOn: 'Now playing on'
+}
+
+type ShareToStoryStickerProps = {
+  track: Pick<Track, 'cover_art_sizes' | 'cover_art' | 'owner_id' | 'title'>
+  user?: Pick<User, 'creator_node_endpoint'>
+  artist: Pick<User, 'user_id' | 'name' | 'is_verified'>
+  style?: StyleProp<ViewStyle>
+  /** Called once the image loads successfully */
+  onLoad: () => void
+}
+
+const useStyles = makeStyles(({ palette }) => ({
+  container: {
+    width: 264,
+    backgroundColor: palette.staticWhite,
+    paddingTop: 16,
```

Comment:
i think we are trying to use `spacing(4)` in these cases, spacing is an arg in makeStyles